frontend/src/components/dashboard/DashboardBlocks.jsx

```diff
@@ -8,90 +8,124 @@ import IconBlocks from "../../../public/static/images/icon-blocks.svg";
 
 import DashboardBlocksBlock from "./DashboardBlocksBlock";
 
-export default ({ blocks }) => (
-  <>
-    <Row>
-      <Col xs="1">
-        <IconBlocks className="dashboard-blocks-icon" />
-      </Col>
-      <Col className="dashboard-blocks-title">
-        <h2>Recent Blocks</h2>
-      </Col>
-    </Row>
-    <Row>
-      <Col xs="1">
-        <div className="dashboard-blocks-hr-parent">
-          <div className="dashboard-blocks-hr" />
-        </div>
-      </Col>
-      <Col>
-        <Row className="gutter-4">
-          {blocks.slice(0, 8).map(block => (
-            <DashboardBlocksBlock
-              key={block.hash}
-              blockHash={block.hash}
-              blockHeight={block.height}
-              blockTimestamp={block.timestamp}
-              transactionsCount={block.transactionsCount}
-            />
-          ))}
+import BlocksApi from "../../libraries/explorer-wamp/blocks";
+import FlipMove from "react-flip-move";
+
+export default class extends React.Component {
+  state = {
+    blocks: this.props.blocks
+  };
+
+  componentDidMount() {
+    this.regularFetchInfo();
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this.timer);
+    this.timer = null;
+  }
+
+  fetchInfo = async () => {
+    const blocks = await new BlocksApi()
+      .getLatestBlocksInfo(8)
+      .catch(() => null);
+    this.setState({ blocks });
+  };
+
+  regularFetchInfo = async () => {
+    await this.fetchInfo();
+    if (this.timer !== null) {
+      this.timer = setTimeout(this.regularFetchInfo, 10000);
+    }
+  };
+  render() {
+    const { blocks } = this.state;
+    return (
+      <>
+        <Row>
+          <Col xs="1">
+            <IconBlocks className="dashboard-blocks-icon" />
+          </Col>
+          <Col className="dashboard-blocks-title">
+            <h2>Recent Blocks</h2>
+          </Col>
         </Row>
         <Row>
-          <Col xs="6">
-            <Link href="blocks">
-              <a className="dashboard-footer">View All</a>
-            </Link>
+          <Col xs="1">
+            <div className="dashboard-blocks-hr-parent">
+              <div className="dashboard-blocks-hr" />
+            </div>
+          </Col>
+          <Col xs="10">
+            <FlipMove duration={1000} staggerDurationBy={0}>
+              {blocks.map(block => (
+                <DashboardBlocksBlock
+                  key={block.hash}
+                  blockHash={block.hash}
+                  blockHeight={block.height}
+                  blockTimestamp={block.timestamp}
+                  transactionsCount={block.transactionsCount}
+                />
+              ))}
+            </FlipMove>
+            <Row>
+              <Col xs="6">
+                <Link href="blocks">
+                  <a className="dashboard-footer">View All</a>
+                </Link>
+              </Col>
+            </Row>
```

Comment:
This doesn't have a two-column layout on a big screen now. Let's try `md="6"` alongside with `xs="12"` for the `Col`

Reply:
![image](https://user-images.githubusercontent.com/304265/70279451-32669f00-17bf-11ea-8683-f2644378d903.png)